docs/ml/deploy.md

```diff
@@ -13,19 +13,15 @@ icon: "/services/icons/ml.svg"
 # SME: Aaron Casas
 ---
 
-Once you have [trained](/ml/train-model/) or [uploaded](/ml/upload-model/) your model, the Machine Learning (ML) model service allows you to deploy machine learning models to your machine.
+The Machine Learning (ML) model service allows you to deploy machine learning models to your machine.
 
-You can use the following built-in model:
+You can use the following built-in model of the service:
```

Comment:
@npentrel Added this phrasing here and below. it makes it a bit clunkier but I felt it was super confusing w/ service models vs ml models across this page